src/wallet/wallet2.cpp

```diff
@@ -3550,8 +3613,94 @@ std::vector<wallet2::pending_tx> wallet2::create_transactions(std::vector<crypto
 
 void wallet2::get_outs(std::vector<std::vector<tools::wallet2::get_outs_entry>> &outs, const std::list<size_t> &selected_transfers, size_t fake_outputs_count)
 {
-  LOG_PRINT_L2("fake_outputs_count: " << fake_outputs_count);
   outs.clear();
+  
+  if(m_light_wallet && fake_outputs_count > 0) {
```

Comment:
No, I don't want that. Is there a valid scenario where fake_outputs_count = 0? Or is that a leftover from before last hf?